### DISCUSSION TITLE
Open collective project and goals: Linux/macOs support

### DISCUSSION THREAD

**Discussion by Aggror** [stride3d/stride#1202] [Contributors]
A rather large topic that require a lot of work is the the support for Linux and/or macOs. When worked out in more detail, it can be used in the plan for requesting an Epic grant.

1. What needs to be done?
-  Runtime/3d
-  Build tools (able to msbuild a Stride project and have the asset compiler work on it) requirement:  FBX/Assimp
-  Editor 

2.  How much time would this cost (estimate, weeks, months)

3.  (Optional) Who could potentially make this?

https://opencollective.com/stride3d/projects/linux-runtime-support

**Comment by Eideren** [comment]
Relevant issue https://github.com/stride3d/stride/issues/360

**Comment by manio143** [comment]
From community meeting - top things:
1. Get things running on Linux outside of the default scene I used for my testing, to verify there's no hidden bugs in the graphics APIs platform dependent
2. Fix OpenGL as not everyone will want to use Vulkan on Linux
3. Build+Deploy scenario - make it possible to just zip the bin folder from a Linux project and run it on a Linux machine (I remember there was something missing here)
4. Provide documentation for users to build from windows and deploy on linux
